pkg/api/v1alpha1/cluster_types.go

```diff
@@ -1156,6 +1159,27 @@ func (c *Cluster) ControlPlaneAnnotation() string {
 	return controlPlaneAnnotation
 }
 
+// DisableControlPlaneIPCheck sets the `skip-ip-check` annotation on the Cluster object.
+func (c *Cluster) DisableControlPlaneIPCheck() {
+	if c.Annotations == nil {
+		c.Annotations = make(map[string]string, 1)
+	}
+	c.Annotations[skipIPCheckAnnotation] = "true"
+}
+
+// SkipControlPlaneIPCheck checks it the `skip-ip-check` annotation is set on the Cluster object.
+func (c *Cluster) SkipControlPlaneIPCheck() bool {
+	if s, ok := c.Annotations[skipIPCheckAnnotation]; ok {
+		return s == "true"
+	}
+	return false
+}
+
+// SkipIPCheckAnnotation returns the `skip-ip-check` annotation as a string.
+func (c *Cluster) SkipIPCheckAnnotation() string {
```

Comment:
This isn't being used in the code anywhere so I can remove it. I added this exported method in case the annotation needs to be accessed outside of the v1alpha1 package.